### DISCUSSION TITLE
App Reiniger

### DISCUSSION THREAD

**Discussion by Ups53562** [d4rken-org/sdmaid-se#1299] [Q&A]
Hallo was genau löst die Funktion App Reiniger aus. Ich zögere die Funktion zu nutzen wenn ich bei Mails z.B. sehe dort sind 200 MB löschbar und dei anderen Apps sogar noch mehr?

**Comment by d4rken** [comment]
Der AppReiniger löscht alle Arten von Caches. Was das genau bedeutet hängt von der App selber ab, also wo und wie die ihre Daten speichert.

Du kannst in SD Maid immer weiter in die Details rein klicken. Wenn dort nur "Standard Default Caches" aufgelistet sind, kann man diese bedenkenlos leeren. Das würde das System auch machen bei Bedarf.
